src/style.css

```diff
@@ -0,0 +1,123 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", Arial, sans-serif;
+  color: #050b35;
+}
+
+.container {
+  width: 200px;
+}
+
```

Comment:
do not set sizes to blocks, use mix where necessary, and move all external styles under element selector.